signatures/xmss/xmss-sha2/param/xmss-sha2-10-256.yaml

```diff
@@ -0,0 +1,12 @@
+name: XMSS-SHA2_10_256
+security level:
+    classical: 256
+    quantum: 256
+number of operations: 2^10
+failure probability: 0
+sizes:
+    sk: 3116
+    pk: 68
+    ct/sig: 2500
+    msg: 256
+    comment: "value for `sk` depends vastly on the implementation, and can be balanced against signing time. The given value is when using `treehash` from BDS08, estimated. Source: https://crypto.stackexchange.com/a/77292 . Absolute minimum is 132 bytes."
```

Comment:
The [original paper](https://eprint.iacr.org/2011/484.pdf) discusses some of the trade-offs around private key size. That's maybe a better reference.